src/models/transactions/offerCreate.ts

```diff
@@ -0,0 +1,80 @@
+import { ValidationError } from "../../common/errors";
+import { Amount, IssuedCurrencyAmount } from "../common";
+import { CommonFields, verifyCommonFields } from "./common";
+
+export enum OfferCreateFlag {
+    tfPassive = 0x00010000,
+    tfImmediateOrCancel = 0x00020000,
+    tfFillOrKill = 0x00040000,
+    tfSell = 0x00080000,
+}
+
+export enum AccountSetFlag {
+    asfAccountTxnID = 5
+    .
+}
+export interface AccountSet extends CommonFields {
+    TransactionType: "OfferCreate";
+    Flags: undefined
+    setFlag?: number | AccountSetFlag
+}
```

Comment:
Why is this here?

Reply:
Left over from paired programing w/ Mukul, removing it